test/load/allocation/README.md

```diff
@@ -0,0 +1,10 @@
+This test creates a proper amount of GameServers at the initial step, switch them to Allocated state and finally shut them down (`automaticShutdownDelayMin` flag in a simple-udp).
```

Comment:
I'm not sure i understand what "proper amount" means here? Can we rephrase?

Added a top level contention for the test -- not sure I got it 100% right, please review.

```suggestion
This is a load test to determine Allocation QPS over time against a set of GameServers that are constantly being shutdown after a period.

This test creates a proper amount of GameServers at the initial step, switch them to Allocated state and finally shut them down after a set period (`automaticShutdownDelayMin` flag in a simple-udp).
```

Reply:
Updated